tests/system_health/mellanox/mellanox_device_mocker.py

```diff
@@ -15,7 +16,11 @@ def mock_asic_temperature(self, value):
         self.helper.mock_value(AsicData.TEMPERATURE_FILE, str(value))
 
     def get_asic_temperature_threshold(self):
-        value = self.helper.read_value(AsicData.THRESHOLD_FILE)
+        threshold_file = '/run/hw-management/thermal/asic_temp_emergency'
+        hw_mgmt_version = get_hw_management_version(self.helper.dut)
```

Comment:
I think that it will be more python and more clean to write the logic as follows:

if hw-mgmt-version < X
   file = old
else:
   file = new

This reveals the intention of the if statement

Reply:
Discussed with Nana, and would not change the if statement, for it's compatible with the style of the script.